### PR TITLE
Add post about pushing images to VMWare

### PR DESCRIPTION
**Pull request by stefwalter**
Depends on the following pull request being merged first:

 * [x] #10

### CODE REVIEW

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+First install Composer:
+
+    $ sudo yum install lorax-composer cockpit-composer cockpit composer-cli
+
+Next make sure to turn off [SELinux](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/5/html/deployment_guide/ch-selinux) on the system. Lorax doesn't yet work properly with
+SELinux running, as it installs an entire OS image in an alternate directory:
+
+    $ sudo setenforce 0
+    $ sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/' /etc/selinux/config
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Permissive

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Now start ```lorax-composer``` system service:
+
+    $ sudo systemctl enable --now lorax-composer
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: lorax-composer.socket

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+defined. This blueprint describes what goes into the image. For the purposes of this
+example we'll use the ```example-http-server``` blueprint, which builds an image that
+contains a basic HTTP server.
+
+We run the following command to start a compose. Notice that we pass the image type
+of ```vhd``` which indicates we want an image appropriate for pushing to
+*Azure* in the *Virtual Hard Disk* format.
+
+    $ sudo composer compose start example-http-server vmdk
+    XXXX
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Fill in XXXX with a UUID

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Now check the status of the compose like this:
+
+    $ sudo composer compose status
+    XXX
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Fill in XXX with a status output

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    ...
+
+When it's done you can download the resulting image into the current directory:
+
+    $ sudo composer compose image 25ccb8dd-3872-477f-9e3d-c2970cd4bbaf
+    XXX
+
+## Pushing and using the image
+
+XXXX
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Appears to be unfinished :)

**Review by bcl — CHANGES_REQUESTED** [review]
Same small comments, and I guess this PR is a work in progress still.

**Comment by bcl** [comment]
And a missing image:
/images/cockpit-composer-download-vmware.png

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+It's running on port ```9090``` on the VM that you're running Composer in. Use any admin
+or ```root``` Linux system credentials to log in.  Select the *Image Builder* tab.
+
+![Cockpit Composer](/images/cockpit-composer-main.png)
+
+We first have to have a *blueprint* defined. This blueprint describes what goes into the image.
+For the purposes of this example we'll use the ```example-http-server``` blueprint, which
+builds an image that contains a basic HTTP server.
+
+Click on the *Create Image* button and choose *Azure* from the dropdown
```
Comment (VladimirSlavik) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Azure?

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+to choose the *Image Type*:
+
+![Create Image AMI](/images/cockpit-composer-create-vmware.png)
```
Comment (VladimirSlavik) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md]: AMI?

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 1 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Fixed.

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 2 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Fixed.

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 3 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Completed.

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 4 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Completed

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 7 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md]: Fixed.

**Comment by stefwalter** [comment]
@VladimirSlavik  @bcl I've completed this document. There is currently no way to use Composer UI (in Cockpit) to accomplish this task (due to the need to add a root password to the image). So I've removed that section.

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+    $ composer-cli blueprints save example-http-server
+
+Add the following lines to the end of the ```example-http-server.toml``` file to set
+the initial root password to ```foobar```. You can also use a crypted password string.
+
+    [[customizations.user]]
+    name = "root"
+    password = "foobar"
+
```
Comment (bcl) [inline _posts/2018-10-08-Pushing-Images-to-VMWare.md:62]: We should probably also tell them they can set a ssh key here with key = "..."

**Review by bcl — APPROVED** [review]
Looks fine other than the 1 suggestion.

File: _posts/2018-10-08-Pushing-Images-to-VMWare.md (hunk 8 above)
Comment (stefwalter) [reply _posts/2018-10-08-Pushing-Images-to-VMWare.md:62]: Done.

**Comment by stefwalter** [comment]
Rebased.

**Pull request merged**